test/kafo_wizards/entries/netmask_test.rb

```diff
@@ -31,7 +31,14 @@
       err = proc {
         netmask.validate('127001/56')
       }.must_raise KafoWizards::ValidationError
-      err.message.must_equal "127001/56 is not valid netmask (invalid length)"
+
+      expected_error_message = if Gem::Version.new(RUBY_VERSION) >= Gem::Version.new('3.1.2')
```

Comment:
Interesting version number, because these kind of changes are quite unusual in patch versions. That made me wonder where it was changed and it's https://github.com/ruby/ruby/commit/54ee22be5308ee958a73bd546bc867d2337d74d8

You can see that's part of 3.1.0
```suggestion
      expected_error_message = if Gem::Version.new(RUBY_VERSION) >= Gem::Version.new('3.1')
```